### PR TITLE
Add s3 signed url service

### PR DESCRIPTION
**Pull request by Ramias1**
Hopefully I did this pull request correctly.

### CODE REVIEW

**Comment by Ramias1** [comment]
will try to sync first and re-pull

**Comment by Ramias1** [comment]
Sorry I am not a GitHub power user.  Looks like my code is slightly out of sync with work you did recently.  Hopefully you can merge this in.  Let me know if you want me to resync first and I'll try to figure that out.

File: README.md [hunk 1]
```diff
@@ -62,6 +62,19 @@ Example data for service call:
 }
 ```
 
+### Sign URL Service
+The S3 entity exposes a `signurl` service fpr generating pre-signed URLs with a defined validity period for accessing content already stored in S3 with a URL.  Run this action after you call the S3 copy service.  This service generates an event of type s3_signed_url which you can use as a trigger in a subsequent automation.  The event data returns a key-value pair of URL and the pre-signed URL.
```
Comment (robmarkcole) [inline README.md:66]: typo: fpr

**Comment by robmarkcole** [comment]
And the conflict there

**Comment by Ramias1** [comment]
Ok I think I did this write this time.  Thanks for the patience.

**Comment by robmarkcole** [comment]
Fantastic @Ramias1

**Pull request merged**